pkgs/applications/misc/hellwal/default.nix

```
@@ -0,0 +1,43 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  makeWrapper,
+}:
+
+let
+  pkgs = import <nixpkgs> {};
+in
+pkgs.stdenv.mkDerivation rec {
+  pname = "hellwal";
+  version = "1.0.2";
+
+  src = pkgs.fetchurl {
+    url = "https://github.com/danihek/hellwal/archive/v1.0.2.tar.gz";
+    sha256 = "sha256-7/ldegQD/AbVeRfktNTb+Jem8DABBFks+VvntUyDvJA=";
+  };
```

Use `fetchFromGitHub` instead, and prefer `hash` over `sha256`
```suggestion
  src = fetchFromGitHub {
    owner = "danihek";
    repo = "hellwal";
    tag = "v${finalAttrs.version}";
    hash = "sha256-7/ldegQD/AbVeRfktNTb+Jem8DABBFks+VvntUyDvJA=";
  };
```